hazelcast/src/test/java/com/hazelcast/test/DockerTestUtil.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.test;
+
+public class DockerTestUtil {
+    public static boolean dockerDisabled() {
+        return System.getProperties().containsKey("disable.docker.tests");
```

Comment:
We usually prefix the properties with `hazelcast.`, also for test properties - see e.g. `HazelcastTestSupport`, something like `hazelcast.docker.tests.enabled`, with default value `true`, because of `dockerDisabled` -> `dockerEnabled` rename.

Reply:
I'm fine with adding `hazelcast.` prefix. 

My reasoning was that it should be clear that docker tests are enabled by default. Having `hazelcast.docker.tests.enabled` suggests that they need to be explicitly turned on.

Reply:
Hmm, I see your point - it makes it easier to read here:
```
    public static boolean dockerEnabled() {
        return Boolean.parseValue(System.getProperty("hazelcastr.docker.tests.enabled", "true"));
    }

    public static boolean dockerDisabled() {
        return System.getProperties().containsKey("hazelcastr.docker.tests.disabled");
    }
```

but here imho the _enabled_ version is much easier to process - this will be used in most tests.
```
    assumeFalse(DockerTestUtil.dockerDisabled());
    assumeTrue(DockerTestUtil.dockerEnabled());
```

Up to you ...

Reply:
What if we would add this to `DockerTestUtil` and use it in tests?
```java
    public static void assumeDockerEnabled() {
        assumeFalse(dockerDisabled());
    }
```

Reply:
@frant-hartm Actually, I've removed the double negative but left the `hazelcast.docker.tests.disabled` property. Are you OK with that?